pkg/sync/expand/cycle.go

```diff
@@ -0,0 +1,238 @@
+package expand
+
+import (
+	mapset "github.com/deckarep/golang-set/v2"
+	"reflect"
```

Comment:
These import statements anger the linter, which wants go packages put above libraries. If you run `make lint` locally you should see the errors. I think gofmt will fix them. I have that hooked up in my editor to reformat on save.